jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaPanel.form

```diff
@@ -121,7 +121,7 @@
           <grid row="7" column="1" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="0" fill="1" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
-          <enabled value="false"/>
+          <enabled value="true"/>
```

Comment:
They are supposed to be disabled due to the values are loaded in from the network, so they should not be enabled till it loads.

Reply:
Reverted this change.